practiceCourseGenerator/README.md

```diff
@@ -1,39 +1,52 @@
-# Canvas Practice Course Generator
-
-Description: Python script for generating zip file that can be used to create 
-practice courses for instructors in Canvas.
-
-## Installation:
-
-1. Install Vagrant <http://www.vagrantup.com/downloads>
-2. Start virtual server `vagrant up`
-3. Login into vertual server: `vagrant ssh`
-4. cd \vagrant
-5. Run application: `python practiceCourseGenerator.py . .`
-
-## Running application
-
-Input: 
-
-1. Canvas Practice Course Training List (Responses).csv - gathered from Google Drive
-2. propertiesProd.json -  holds custom properties
-3. storage.json - holds information to access google drive
-4. tl_client_secret.json - holds information to access google drive
+# Canvas Practice Course Generator
+
+Description: Python script for generating zip file that can be used to create practice courses for instructors in Canvas.

```

Comment:
From the cantankerous developer dept.: I think of this and other Python code as programs, which are steps above mere scripts.